### PR TITLE
form:adding form test case scenario

### PR DESCRIPTION
**Pull request by gitcliff**
link:https://github.com/IsantePlus/isanteplus-qaframework/issues/21

### CODE REVIEW

**Comment by mozzy11** [comment]
@gitcliff  can you resolve the compilation errors at 

```
Error:  COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
Error:  /home/runner/work/isanteplus-qaframework/isanteplus-qaframework/src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/page/ClinicianFacingPatientDashboardPage.java:[100,16] cannot find symbol
  symbol:   class PatientFormHistoryPage
  location: class org.openmrs.contrib.isanteplus.qaframework.automation.page.ClinicianFacingPatientDashboardPage
```

File: src/features/isanteplus/Form.feature [hunk 1]
```diff
@@ -17,7 +17,7 @@ Feature: Adding/Editing forms and Patient Cover Sheet
   Then The form should be listed under Historique des formulaires
   Examples:
    |searchText |
-   | sharif2   |
+   | cliff     |
```
Comment (sherrif10) [inline src/features/isanteplus/Form.feature:20]: Makes sense @gitcliff  . Well done on this

File: src/features/isanteplus/addPatient.feature [hunk 2]
```diff
@@ -24,5 +24,4 @@ Feature: Adding a Patient
     |sharif1   |sharif1  |Male   |10  |STCODE3 |HAITI101 | haiti   |
     |moses3    |moses3   |Male   |17  |STCODE55|HAITI103 | haiti   |
     |sharif2   |sharif2  |Male   |12  |STCODE4 |HAITI102 | haiti   |
-    |cliff     |gita     |Male   |13  |STCODE5 |HAITI104 | haiti   |
```
Comment (sherrif10) [inline src/features/isanteplus/addPatient.feature]: You can also make this name available, Also if you use any name , its better to run it once for the system to be able to allocate itself once

**Comment by sherrif10** [comment]
:face_with_spiral_eyes:  Man thnks for the huste

**Comment by mozzy11** [comment]
Looks like this PR breaks the build , since i merged it

**Comment by mozzy11** [comment]
@gitcliff , do you want to fix the build failure ,why it fails on merging this so that i dont revert the entire PR

**Pull request merged**